packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.styles.ts

```diff
@@ -164,4 +165,22 @@ export const styles: ElementStyles = css`
     ${typographyBody1StrongStyles}
     color: ${colorNeutralForeground1};
   }
-`;
+  @media (forced-colors: active) {
+    .bar-label {
```

Comment:
Can these styles be included within `forcedColorsStylesheetBehavior`?

Reply:
Done

Reply:
The media query is redundant as it is already managed by `forcedColorsStylesheetBehavior`.
See this: https://github.com/microsoft/fluentui/blob/85a3aa172ee82f8477eeb4f2ae5a85f52920b82a/packages/web-components/src/utils/behaviors/match-media-stylesheet-behavior.ts#L171

Reply:
Okay